src/binwalk/modules/extractor.py

```diff
@@ -981,10 +981,10 @@ def shell_call(self, command):
             rval = subprocess.call(shlex.split(command), stdout=tmp, stderr=tmp)
 
         # A true child process should exit with the subprocess exit value
-        if child_pid is 0:
+        if child_pid == 0:
             sys.exit(rval)
         # If no os.fork() happened, just return the subprocess exit value
-        elif child_pid is None:
+        elif child_pid == None:
```

Comment:
Did you check the warnings after installing this version? `None` comparisons should use `is`, this change introduces a new warning.